rules/eslint/style/walmart.js

```diff
@@ -102,6 +118,8 @@ module.exports = {
     "space-before-blocks": [2, "always"],
     // require or disallow space before function opening parenthesis
     "space-before-function-paren": [2, {anonymous: "always", named: "never" }],
+    // equire a space before certain keywords (fixable)
+    "space-before-keywords": 0,
```

Comment:
:+1: This is a good one to get dialed in since in the latest ESLint (and much more so in 2.0 which is already in alpha) this rule can be auto fixed! 

http://eslint.org/docs/rules/space-before-keywords